fastn/src/main.rs

```diff
@@ -61,6 +53,10 @@ async fn fastn_core_commands(matches: &clap::ArgMatches) -> fastn_core::Result<(
     use colored::Colorize;
     use fastn_core::utils::ValueOf;
 
+    if matches.subcommand_name().is_none() {
```

Comment:
This change was required as call to `fastn_core::Config::read()` made on [line 84](https://github.com/fastn-stack/fastn/pull/1439/files#diff-0d3e40f8767e766fe81a191eb5706ed7afa961d1ad16fc4bc47d103c5276d823L87) errors when used outside of a package dir even when this is not required when no subcommand is passed in the cli.